Assets/_PlatformerDevelopment/Tests/SnapshotTestCase.cs

```diff
@@ -0,0 +1,133 @@
+﻿//-----------------------------------------------------------------------
+// <copyright file="SnapshotTestCase.cs" company="PERIGON GAMES">
+//     Copyright (c) PERIGON GAMES. 2020 All Rights Reserved
+// </copyright>
+//-----------------------------------------------------------------------
+using System.Diagnostics;
+using System.IO;
+using NUnit.Framework;
+using UnityEditor;
+using UnityEngine;
+
+namespace Tests
```

Comment:
This helps with taking a snapshot of the scene and storing the snapshots under
"Assets/_PlatformerDevelopment/Tests/ScreenShots/"